Exercises/STAM_Exercise.py

```diff
@@ -0,0 +1,59 @@
+__author__ = "Sol"
```

Comment:
any python file should start with one line doc that explain what the purpose of the file